docs/jwst/ami/main.rst

```diff
@@ -3,155 +3,12 @@ Tasks in the Package
 The Aperture Masking Interferometry (AMI) package currently consists
```

Comment:
Terminology question: Here, and in other locations, steps are being referred to as "tasks". Should they not be referred to as "steps" directly?